lib/iris/_representation/cube_summary.py

```diff
@@ -265,12 +265,11 @@ def __init__(self, title, cell_methods):
         self.values = []
         self.contents = []
         for method in cell_methods:
-            name = method.method
-            # Remove "method: " from the front of the string, leaving the value.
-            value = str(method)[len(name + ": ") :]
-            self.names.append(name)
+            index = str(cell_methods.index(method))
```

Comment:
In the line above (it won't let me make a direct suggestion!), use:

```python
for index, method in enumerate(cell_methods):
```

Then you won't need to derive `index` after the fact.